x-pack/plugins/ml/public/file_datavisualizer/components/fields_stats/field_stats_card.js

```diff
@@ -28,7 +28,7 @@ export function FieldStatsCard({ field }) {
           <div
             className={`ml-field-title-bar ${type}`}
           >
-            <FieldTypeIcon type={type} />
+            <FieldTypeIcon type={type} ariaSuffix={field.name} />
             <div className="field-name">{field.name}</div>
```

Comment:
As above, I wonder if a better way of achieving this is to wrap the `div` below with an `EuiKeyboardAccessible` element?